system-x/services/jms/ibm-mq/src/main/java/software/tnb/jms/ibm/mq/service/IBMMQ.java

```diff
@@ -28,15 +28,6 @@ public abstract class IBMMQ extends ConfigurableService<IBMMQAccount, Connection
     public static final String MQSC_COMMAND_FILE_NAME = "99-tnb.mqsc";
     public static final String SSLCIPHERSUITE = "TLS_RSA_WITH_AES_128_CBC_SHA256";
 
-    public IBMMQValidation validation() {
```

Comment:
is it ok the remove it? I know it was added to openshift, what about standalone test?

Reply:
the method was added to standalone as well

Reply:
omg I am sleeping